homeassistant/components/buienradar/weather.py

```diff
@@ -201,7 +201,7 @@ def forecast(self):
             # keys understood by the weather component:
             condcode = data_in.get(CONDITION, []).get(CONDCODE)
             data_out = {
-                ATTR_FORECAST_TIME: data_in.get(DATETIME),
+                ATTR_FORECAST_TIME: data_in.get(DATETIME).isoformat(),
```

Comment:
If the key would be missing we'd get an error here. If we know that the key is not missing we should use`dict[key]` instead of `dict.get(key)`.